tools/integration_tests/readwrite/readwrite_test.go

```diff
@@ -83,14 +83,23 @@ func mountGcsfuse() error {
 	return nil
 }
 
-func umount(dir string) error {
-	umount, err := exec.LookPath("umount")
+func unMount() error {
+	fusermount, err := exec.LookPath("fusermount")
 	if err != nil {
-		return fmt.Errorf("cannot find umount: %w", err)
+		return fmt.Errorf("cannot find fusermount: %w", err)
 	}
-	cmd := exec.Command(umount, mntDir)
+	cmd := exec.Command(fusermount, "-uz", mntDir)
 	if _, err := cmd.CombinedOutput(); err != nil {
-		return fmt.Errorf("umount error: %w", err)
+		return fmt.Errorf("fusermount error: %w", err)
+	}
+	return nil
+}
+
+func clearKernelCache() error {
+	// sudo permission is required to clear kernel page cache.
+	cmd := exec.Command("sudo", "sh", "-c", "echo 3 > /proc/sys/vm/drop_caches")
```

Comment:
This may not be present on all the platforms. Should we do a check if this exists and only write if it does?